bolt-design-system/bolt
# Fix: bolt-share spacing bug and addition of align prop

## Jira

http://vjira2:8080/browse/BDS-717

## Summary

Fixes the `bolt-share`'s spacing bug in FF and adds an align prop for horizontal alignment.

## Details

Removed some unnecessary margin CSS to fix the FF spacing bug. Align prop is added since this is a block level component similar to `bolt-pagination`, the options should be consistent.

Also changed all the docs to use some common variables while still retain the `include "@bolt-components-share/share.twig"` on every doc page. Makes it less confusing when someone is looking at the code panel in Pattern Lab.

## How to test

Pull down and run the branch locally. Check all the share demos in all browsers. Make sure spacing is the same throughout, and align prop works as expected.
